site/content/en/docs/Guides/Client SDKs/godot.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Godot Game Server Client SDK"
+linkTitle: "Godot"
+date: 2021-04-15T10:17:50Z
```

Comment:
Please set this to 2021-04-27 which is when the 1.14 release will be cut so that it doesn't get published yet.